snowflake/datadog_checks/snowflake/check.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, *args, **kwargs):
         self.errors = []
         for mgroup in self._config.metric_groups:
             try:
+                if self._config.use_deprecated_aggregation:
```

Comment:
Can we add a test for when it's disabled? And a unit test to ensure these queries look right when the option is enabled?